ui/src/containers/ExportDownloadManager/index.js

```diff
@@ -17,18 +19,34 @@ class ExportDownloadManager extends Component {
   render = () => {
     const { models, isLoading } = this.props;
     if (isLoading) return (<Spinner />);
-
     return (
       <div>
-        { models.map((model, id) =>
-          <DownloadListItem key={id} model={model} deleteModel={this.props.deleteModel} />
+        { 
+          models.map((model, id) =>
+          <DownloadListItem
+            key={id}
+            model={model}
+            exporationExport={this.props.organisation.getIn(['settings', 'EXPIRE_EXPORTS'])}
+            deleteModel={this.props.deleteModel} />
         ).valueSeq() }
       </div>
     );
   }
 }
 
 export default compose(
-  withProps({ sort: new Map({ time: -1, _id: 1 }) }),
-  withSchema('download')
+  withProps({
+    sort: new Map({ time: -1, _id: 1 }),
+    schema: 'organisation'
```

Comment:
Bit confusing to set to the schema to `organisation` here. Probably set the schema in the connect so that it's set next to the orgId. Then move the `withProps` immediately above the `withSchema` so that it's obvious it's for the `download` models.

Reply:
I moved `sort`